src/logger.rs

```diff
@@ -22,6 +22,7 @@ pub fn setup(is_debug: bool) -> Result<(), log::Error> {
         })
         .level(log::LevelFilter::Off)
         .level_for("panic", log::LevelFilter::Error)
+        .level_for("iced_wgpu", log::LevelFilter::Info)
```

Comment:
Included so we can see if `wgpu` is initialized in the logs. I've confirmed flatpak + wgpu works fine! The tiny-skia override also works via ICED_BACKEND var